app/js/question_controller.js

```diff
@@ -109,31 +139,49 @@
         answers[i].created = Utils.time_since(new Date(created));
       }
 
+      add_thread_header(question);
       var html = nunjucks.render('thread.html', {
+        user: user,
         results: answers
       });
-      document.getElementById('comment-list').innerHTML = html;
+      var list = document.getElementById('comment-list');
+      list.insertAdjacentHTML('beforeend', html);
     });
   }
 
   var QuestionController = {
     init: function() {
-      var form = document.getElementById('question_form');
-      form.addEventListener('submit', submit_comment);
+      var question_view = location.search ? true : false;
+
+      // if this is a question view, not arrived at from clicking/tapping
+      // ask a question, hide the introduction section.
+      if (question_view) {
+        document.getElementById('thread-introduction').classList.add('hide');
+      }
```

Comment:
I think keeping this logic inside `show_question` makes more sense because the "if" is already included. It explains more clearly that "when we show a question, we hide the introduction".